ledger/tasks.py

```diff
@@ -10,7 +10,9 @@ def record_transaction(data):
     Record taco transactions.
     """
     giver = data.get('giver_id')
-    tacos_given_today = TacoLedger.objects.filter(giver=giver, timestamp__date=date.today().day).aggregate(Sum('amount'))
+    tacos_given_today = TacoLedger.objects.filter(giver=giver, timestamp__day=date.today().day).aggregate(Sum('amount')).get('amount_sum')
+    if not tacos_given_today:
+        tacos_given_today = 0
```

Comment:
What would you think about using [Coalesce](https://docs.djangoproject.com/en/5.0/ref/models/database-functions/#coalesce) here instead of a conditional after the aggregation?

Something like this: `TacoLedger.objects.filter(giver=giver, timestamp__day=date.today().day).aggregate(amount_sum=Coalesce(Sum('amount'),0))['amount_sum']`

Reply:
I don't know what coalesce does so I'll have to research it to figure out what I think!